### DISCUSSION TITLE
Welcome to Themix Discussions!

### DISCUSSION THREAD

**Discussion by actionless** [themix-project/themix-gui#347] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bllooi** [comment]
I was wondering what font you were using in that readme! I like it alot and want to use it with the oomox theme.

**Reply by actionless** [reply]
if you asking about this screenshot then it's Fantasque Sans Mono:

![](https://raw.githubusercontent.com/themix-project/oomox/master/screenshots/pokedex_dawn.png)
